recipes/cuda-ctadvisor/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set cuda_version = "13.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}  # [aarch64 and arm_variant_type == "sbsa"]
+{% set platform = "linux-aarch64" %}  # [aarch64 and arm_variant_type == "tegra"]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+{% set soname = version.split(".")[0] %}
+
+{% set sha256 = "b2aa722c5fe091574cf7620073b25d7e46337410ff8e1382b326bba795442a9a" %}  # [linux64 and (cuda_compiler_version or "").startswith("13")]
+{% set sha256 = "03e4404e888609bc98f496453896acc34c9ca7bbc39cc32b41d677bcaab71c49" %}  # [aarch64 and (cuda_compiler_version or "").startswith("13")]
+{% set sha256 = "3705255ba8cc32523d2cd1dca93871ce5dceb31fa3c5794639e7dc546b623590" %}  # [win and (cuda_compiler_version or "").startswith("13")]
+
+package:
+  name: cuda-ctadvisor-split
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_ctadvisor/{{ platform }}/cuda_ctadvisor-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  binary_relocation: false
+  skip: true  # [osx or ppc64le or (not (cuda_compiler_version or "").startswith("13"))]
+
+requirements:
+  build:
+    - cf-nvidia-tools 1  # [linux]
+
+outputs:
+  - name: cuda-ctadvisor
+    build:
+      script:
+         - move bin\* %LIBRARY_BIN%  # [win]
+    files:             # [linux]
+      - bin/ctadvisor  # [linux]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}  # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      requires:
+        - patchelf                                        # [linux]
+      files:
+        - test-rpath.sh                                   # [linux]
+      commands:
+        - test -f $PREFIX/bin/ctadvisor                   # [linux]
+        - bash test-rpath.sh                              # [linux]
+        - if not exist %LIBRARY_BIN%\ctadvisor.exe exit 1  # [win]
+
+about:
+  home: https://developer.nvidia.com/cuda-toolkit
+  license_file: LICENSE
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
+  license_url: https://docs.nvidia.com/cuda/eula/index.html
+  summary: ctadvisor is a tool that helps users analyze the compilation time of their CUDA C++ code and provides suggestions to reduce it.
+  description:
+    CUDA Compile Time Advisor (ctadvisor) analyzes trace files containing compilation
+    time information generated by NVCC or NVRTC. ctadvisor identifies compilation
+    bottlenecks that take significant amount of time, for examples, template
+    instantiation and headers processing. And it provides with users with suggestions
+    to reduce compilation time.
```

The last piece here isn't a complete sentence.
```suggestion
    instantiation and headers processing. The tool provides with users with suggestions
    to reduce compilation time.
```

---

Is `provides with users with suggestions` a proper phrase?

---

Good eye. I fixed my suggestion above.